server/server.go

```diff
@@ -338,7 +338,10 @@ func NewServer(userConfig UserConfig, config Config) (*Server, error) {
 		true,
 		projectCmdOutputHandler)
 
-	dd := datadog.NewClient("localhost:8125")
+	dd := datadog.NewClientWith(datadog.ClientConfig{
+		Address:              "localhost:8125",
```

Comment:
Kind of a random comment since I'm reading this months later figuring out how to do distribution metrics... but it looks like veneur runs on 8125 and the datadog agent is on 8126. That's what julien told me today, and a sourcegraph search of config files seems to agree: https://sourcegraph.segment.com/search?q=context:global+veneur%3D&patternType=literal

Reply:
It's probably on 8125 on the atlantis machines... but geez is that confusing if we have two mostly compatible statsd agents running on different machines in different configurations

Reply:
mmm... well `steps.duration` is not a distribution metric in datadog as of today: https://segment.datadoghq.com/metric/explorer?start=1657770180043&end=1658374980043&paused=false#N4IgNghgng9grgFxALlADxQRgDQil3AdwEsATBACwJAoFNiBzCpZTAX11NoDNiA7YgmIw+KUAigAHWihBCAtrQDOtAE7FlIXKtoBHOMoRKUAbVA6lkkSoD63GKvkQWc4opXrNufWo3HkZiB8EIqyPqpQmFogpM4QNkrwqgDGMsggigjqycbeBhGyEABuDMjOkHxCSgB0Sgi0kjWkcKrOwnzVAJwArNIptJXEYLTAAFRsAAQARlATwBQwdWwgbAC6uPaOcJD+gZvy2xBh+ZEr6yB1UMNiIJIQwwj1sqQwDDbJO0rEydFg-LQ2CTSWSJP6kX7-GwkchUdJ8BxOMArTjESyQKCAqRpcD-M5sNhAA

So, I'd say that these `dist_` metrics are being handled by veneur and forwarded as `h` type to datadog.

Reply:
Hey @boggsboggs! In a later commit I switched the namespace to get this working, for some reason this PR didn't work as-is. https://github.com/segmentio/atlantis/commit/e6b721d7b73e4c6fd5a434f08739a5aec07aeaf5 

The correct metric is now `atlantis.dist_duration` which is a distribution metric

Another thing I discovered is that once you submit the metric to datadog, you need to [enable percentile and threshold queries](https://docs.datadoghq.com/metrics/distributions/#enabling-advanced-query-functionality) from the Metrics -> Summary page

And yeah this runs at 8125 on the Atlantis host. We're thinking of adding veneur here as well, in that case we'd run on 8126

Reply:
ah yeah, I see it. Thanks for the original PR illustrating how folks are using distributions, and also the extra explanation.

Interesting that it wasn't working with the `steps.` prefix...